tests/acceptance/expected-failures-with-oc10-server.md

```diff
@@ -0,0 +1,8 @@
+## This file contains the scenarios from web tests that are currently expected to fail on oC10
+
+### [Media Viewer](https://github.com/owncloud/ocis/issues/1106)
+- <div class="text-green">[webUIPreview/imageMediaViewer.feature:81](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L81)</div>
+- [webUIPreview/imageMediaViewer.feature:88](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L88)
+
+### [WebUI Login](https://github.com/owncloud/web/issues/4564)
+- [webUILogin/login.feature:62](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUILogin/login.feature#L62)
```

Comment:
It is a bit painful that `https://github.com/owncloud/web/blob/master/tests/acceptance/features` has to repeat many times. I tried putting a substitution at the end of the document, like:
```
[web-repo-url]: https://github.com/owncloud/web/blob/master/tests/acceptance/features
```

And then trying to put things like 
```
- [webUIPreview/imageMediaViewer.feature:88]([web-repo-url]/webUIPreview/imageMediaViewer.feature#L88)
```

But I don't think that Markdown is smart enough to do that sort of substitution into the URL link.

Also `webUILogin/login.feature:62` gets almost repeated again `webUILogin/login.feature#L62` in a predictable way. But I also don't think that Markdown is fancy enough to be able to reduce that duplication.